scripts/upgrade_exchange.ts

```diff
@@ -0,0 +1,31 @@
+import * as dotenv from "dotenv";
+import { ethers, upgrades } from "hardhat";
+dotenv.config();
+
+/*
+ * this script updates the proxy implementation
+ * of clipto exchange, the proxy address should remain
+ * same.
+ */
+async function main() {
```

Comment:
Ideally scripts like this and the deploy script should be integrated as hardhat tasks, that way the functionality of the deploy/upgrade scripts can be directly tested within hardhat tests.